hbase-server/src/main/java/org/apache/hadoop/hbase/master/region/MasterRegion.java

```diff
@@ -147,9 +147,11 @@ public void update(UpdateMasterRegion action) throws IOException {
     try {
       action.update(region);
       flusherAndCompactor.onUpdate();
-    } catch (WALSyncTimeoutIOException e) {
-      LOG.error(HBaseMarkers.FATAL, "WAL sync timeout. Aborting server.");
-      server.abort("WAL sync timeout", e);
+    } catch (IOException e) {
```

Comment:
Better add some comments here, to summary the discussion on the jira, and also give a pointer to the jira, to let later developpers know why here we will abort for any IOExceptions.

Reply:
Done, thanks!